### DISCUSSION TITLE
CDN Services For Vanilla JS Widget

### DISCUSSION THREAD

**Discussion by codepeach** [vercel/community#481] [Help] [answered]
I am already using Vercel to for the landing page and dashboard of my product. 

Additionally I have an extremely small widget (60kb totally) made with pure vanilla JS and CSS that I need to serve over CDN for others to use on their websites. Would Vercel be the right option for this? 
And if yes, can you point me towards the pricing and limits? For example AWS' Cloudfront costs $0.08/GB for CDN services, but I do not see any such pricing on vercel's website.

**Comment by leerob** [comment, marked as answer]
Hey! You can serve static assets from Vercel, if you would like. [Pricing](https://vercel.com/pricing) and [limits](https://vercel.com/docs/concepts/limits/overview) are here.
